src/charm.py

```diff
@@ -492,6 +535,18 @@ def upgrade_in_progress(self) -> bool:
         """
         return False
 
+    @property
+    def config_server_db(self) -> str:
+        """Fetch current the config server database that this unit is connected to."""
+        if not (
+            config_server_relation := self.model.get_relation(
+                Config.Relations.CLUSTER_RELATIONS_NAME
+            )
+        ):
+            return ""
+
+        return config_server_relation.app.name
+
     # END: properties
 
 
```

Comment:
1. add on top of the file: `from ops.main import main`
2. 2 lines below: call `main`.
(ops.main is not callable, ops.main.main is)